config/routes.rb

```diff
@@ -1,15 +1,21 @@
 Rails.application.routes.draw do
+  root "home#index"
   post "oauth/callback" => "oauths#callback"
   get "oauth/callback" => "oauths#callback"
   get "oauth/:provider" => "oauths#oauth", as: :auth_at_provider
 
-  resources :cards
-  resources :sessions
-  resources :users
+  get "review" => "review#index"
 
-  root "home#index"
   get "login" => "sessions#new"
-  get "review" => "review#index"
   put "review_card" => "review#review_card"
-  post "logout" => "sessions#destroy", as: :logout
+  post "logout" => "sessions#destroy", as: :logout 
+
+  resources :sessions
+  resources :users
+  resources :decks do
+    resources :cards
+    member do
+      put 'set_current', action: :set_current
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.